.github/workflows/tests.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     defaults:
       run:
         shell: bash -l {0}
+        working-directory: ./src/
     strategy:
       fail-fast: false
       matrix:
```

Comment:
Do we want to switch to newer Python versions while we're at it?
(Seems appropriate since this is a CI overhaul PR.)
I'd go for `['3.7', '3.10']`, i.e.,
- use the lowest version still supported
- include the latest version for which all packages are available
  (you can also try `3.11`, but I'm not sure if everything is ready yet; I'd probably wait for this until spring or so)

Reply:
@mbargull In regards with the newer versions there is an effort to overhaul the `conda-build` itself to support the newer version.. it just may be out of the scope for this PR

Reply:
yeah, I'm in favor of doing this in a separate PR

Reply:
Fine by me. It should be a small PR that we can slip on in any time!